packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -180,6 +184,7 @@ export class SpinButton extends React.Component<ISpinButtonProps, ISpinButtonSta
         <KeytipData keytipProps={keytipProps} disabled={disabled}>
           {(keytipAttributes: any): JSX.Element => (
             <div
+              {...nativeProps}
               className={classNames.spinButtonWrapper}
               title={title && title}
```

Comment:
Since `title` is a native prop, this line can be removed.

Reply:
Good catch!